lib/rom/mapper.rb

```diff
@@ -80,11 +91,11 @@ def model
       header.model
     end
 
-    # Process a relation using the transformer
+    # Process a relation using the transformers
     #
     # @api private
     def call(relation)
-      transformer[relation.to_a]
+      transformers.reduce(relation.to_a) { |a, e| e.call(a) }
```

Comment:
This one is rubocop-inspired
https://github.com/bbatsov/ruby-style-guide/issues/196